libs/core/src/lib/calendar/calendar-header/calendar-header.component.html

```diff
@@ -20,6 +20,7 @@
                     [attr.aria-label]="calendarI18nLabels.monthSelectionLabel"
                     [attr.aria-selected]="isOnMonthView()"
                     (click)="processViewChange('month')"
+                    aria-live="polite"
```

Comment:
Having multiple `aria-live` can cause conflicts. There should just be one `aria-live` on a parent element (e.g. the `<header>` would work).